gdal/configure.ac

```diff
@@ -3381,6 +3381,46 @@ if test "$HAVE_OPENJPEG" != "no" ; then
   OPT_GDAL_FORMATS="openjpeg $OPT_GDAL_FORMATS"
 fi
 
+
+
+dnl ---------------------------------------------------------------------------
+dnl Select a Grok Library to use, or disable driver.
+dnl ---------------------------------------------------------------------------
+
```

Comment:
See the "Check if the PDF driver should be built as a plugin" section in that file , and GDALmake.opt.in and GNUmakefile  to add an option to build the driver as plugin

Reply:
It seems to me the pdf driver is the only one where the plugin logic is operated from configure, while the rest seem to follow the existing approach of the grok driver (similar to openjpeg driver).  I was thinking the configure logic might not be appropriate but that person at some point in the future we might want to do a bigger project to make a bunch of drivers support building as plugins in a more automated way.

Reply:
> in the future we might want to do a bigger project to make a bunch of drivers support building as plugins

yes, can make sense. Currently building a driver as a plugin involves some non-trivial steps.

Reply:
Great. So, let's focus on getting this driver merged as currently configured, and then we can work on the plugin option when this is automated.